app/services/flow/flow_state_machine.rb

```diff
@@ -176,6 +176,7 @@ def analytics_properties
         step: current_step,
         step_count: current_flow_step_counts[current_step_name],
         analytics_id: @analytics_id,
+        reproofing: current_user&.decorate&.reproof_for_irs?(service_provider: current_sp).present?,
```

Comment:
is this ticket about reproofing specifically for the IRS? or about reproofing from anywhere, like account recovery?)

Reply:
I like it